Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8192,3 +8192,30 @@ def get_tenant_account_name():
         account_name = "acc_{}".format(tenant_name) if tenant_name != "" else ""
 
     return account_name
+
+def indicators_value_to_clickable(indicators) -> Dict[str, str]:
+    """
+    Function to get the indicator url link for indicators
+    Args:
+        indicators: An indicator or a list of indicators
+
+    Returns:
+        A dictionary. Key is the indicator, and the value is it's url in the server
+
+    """
+    if not indicators:
```

Comment:
Let's check this instead.
```suggestion
    if not isinstance(indicators, (list, dict)):
```